test/testinput/bufr_ncep_adpsfc_nc000000.yaml

```diff
@@ -0,0 +1,188 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+# #
+# # This software is licensed under the terms of the Apache Licence Version 2.0
+# # which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "./testinput/gdas.t12z.adpsfc.nc000000_thinned.tm00.bufr_d"
```

Comment:
There should be no need to pre-split the adpsfc BUFR file if you are manually specifying the subset in all your queries. "NC000000/CLAT" etc. The query itself will filter according to the subset in this case (as long as you specify the specific subset in each query).